go.mod

```diff
@@ -3,18 +3,20 @@ module github.com/mongodb/mongodb-atlas-kubernetes
 go 1.16
 
 require (
+	cloud.google.com/go v0.99.0 // indirect
```

Comment:
are changes to this file necessary?

Reply:
Good catch. I was able to delete this line.